sdk/notificationhubs/notification-hubs/src/auth/sasTokenProvider.ts

```diff
@@ -1,25 +1,43 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-import {
-  AccessToken,
-  NamedKeyCredential,
-  SASCredential,
-  isNamedKeyCredential,
-  isSASCredential,
-} from "@azure/core-auth";
-import { isObjectWithProperties } from "@azure/core-util";
 import { signString } from "./hmacSha256.js";
 
+/**
+ * Represents a named key credential.
+ */
+export interface NamedKeyCredential {
```

Comment:
I'm confused why we're defining these instead of using the version in core-auth?